pages/common/npx.md

```diff
@@ -0,0 +1,15 @@
+# npx
+
+> Execute  npm packages binaries.
+
+- Execute the npm packages binaries:
+
+`npx {{module_name}}`
+
+- Define the package to be installed, then execute the binaries:
+
+`npx -p {{module_name}}`
```

Comment:
I think you missed specifying the package parameter for `-p`.

Reply:
@baixiaoji - Maybe you missed this comment ?

Reply:
you can see the files changed, I especially add some explaination  for your comment.

Reply:
Yes, you changed the description. But you need to update the example to to specify the package name. Shouldn't it be like `npx -p {{package_name}} {{module_name}}` ? That is what I meant.

Reply:
you can see again